torch/_inductor/compile_fx.py

```diff
@@ -252,18 +252,26 @@ def run(new_inputs):
 
 @dynamo_utils.dynamo_timed
 def cudagraphify(model, inputs, static_input_idxs=()):
+    from torch._inductor.cudagraph_trees import (
+        cudagraphify_impl as new_cudagraphify_impl,
+    )
+
+    if config.triton.cudagraph_trees:
```

Comment:
Is it possible to just switch entirely to the new implementation, without the knob here?

Reply:
Not within this PR, after storages pr we could